Source/UI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -7,7 +7,7 @@ public enum PaymentFormFactory {
   internal static var sessionCorrelationID = ""
 
   public static func buildViewController(configuration: PaymentFormConfiguration,
-                                         style: PaymentStyle,
+                                         style: FramesStyle,
```

Comment:
`PaymentStyle` is changed to `FramesStyle` because there's another `PaymentFormStyle` which will be confusing

Reply:
My understanding is, buildViewController was returning back some sort of Payment Form View controller using a `PaymentFormConfiguration` and `PaymentStyle`. Removing the Payment prefix here would be more confusing as everything else will need same prefix like `FramesFormConfiguration`. 

My suggestion would be to change ```
buildViewController(configuration: PaymentFormConfiguration,
                                         style: PaymentStyle,
```
to 
```
buildPaymentFormViewController(configuration: FormConfiguration,
                                         style: FormStyle,
```

Please feel free to add your thoughts.